packages/core/strapi/package.json

```diff
@@ -98,7 +98,6 @@
     "@strapi/permissions": "4.14.0",
     "@strapi/plugin-content-manager": "4.14.0",
     "@strapi/plugin-content-type-builder": "4.14.0",
-    "@strapi/plugin-email": "4.14.0",
```

Comment:
This was causing a circular dependency that caused the NX build to fail. wdyt of this?

The email plugin is only referenced as a string here https://github.com/strapi/strapi/blob/fc86e60e003758b414a1879364d99b1cab94289f/packages/core/strapi/src/core/loaders/plugins/get-enabled-plugins.ts#L32